test/simulation/README.md

```diff
@@ -8,38 +8,63 @@ The simulation tests’ only dependencies are [Docker](https://docs.docker.com/i
 
 ## Usage (via `xud` npm scripts)
 
-Run the tests after building all images:
+#### Run the tests after building all images:
 
 ```bash
 $ npm run test:sim:build
 $ npm run test:sim:run
 ```
 
-Whenever test code is being refactored, one must build the `test` image before re-running the tests so that changes could take effect:
+#### Specific builds
+
+For taking into effect changes in test code, one must build the `test` image before re-running the tests:
```

Comment:
This line is slightly unclear to me. How about "When making changes to the simulation test code, one must build the `test` image before rerunning the tests"